XO.py

```diff
@@ -0,0 +1,157 @@
+from random import shuffle
+
+# size of board
+N = 3
+
+# Utility functions
+
+
+def print_board(board):
+    print()
+    print('Board:')
+    for line_ in board:
+        for item in line_:
+            print(item, end='')
+        print()
+    print()
+
+
+def check_input_validity(input_type, curr_player):
+    not_valid = True

```

Comment:
Always easier to think positive!
`is_valid = False`

Reply:
Done